### DISCUSSION TITLE
game that crashes into world change

### DISCUSSION THREAD

**Discussion by coolsl** [DarkflameUniverse/DarkflameServer#1519] [Q&A] [answered]
I have a bug that prevents me from playing, when I connect to the server and get to the rocket, I build it and the client crashes. When I reconnect, I get to the rocket, then I go to the rocket launcher to change world and it crashes again, I reconnect and it crashes again after I manage to reconnect and I appear at the rocket launcher and it does the same thing over and over again (I'm on an unpacked client) and an error message in the terminal.

**Comment by aronwk-aaron** [comment, marked as answer]
Game crash or kick to login?
please ensure that you have followed the steps in the readme about connecting with a client:
https://github.com/DarkflameUniverse/DarkflameServer?tab=readme-ov-file#allowing-a-user-to-connect-to-your-server
Specifically the part about `UGCUSE3DSERVICES`

**Reply by coolsl** [reply]
it was a client crash and a server kick (the client resets its IP to 127.0.0.1 therefore having to restart it to reconnect) I had forgotten to modify UGCUSE3DSERVICES=7:1, to UGCUSE3DSERVICES=7:0, thank you For your help :D

**Reply by EmosewaMC** [reply]
> the client resets its IP to 127.0.0.1

rename the folder `legouniverse.exe` is in to `client` to fix this issue.
